### PR TITLE
Resources page hourofcode.com

### PR DESCRIPTION
**Pull request by caleybrock**
screen shot 2015-09-24 at 9 45 26 am

Broke resources up into how-to and promote. Created a new nav for these sections.
@tanyaparker @breville 
Currently looking into merging conflicts from staging and the mobile part!

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/views/resources_nav.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+%div{style: "margin-top: 10px;"}
+  %a{href: resolve_url('/promote')}
+    %div.display-link
+      %img.display-img{src: "/images/flyers.png"}
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/views/resources_nav.haml]: you can try requesting a file stored as a png as a jpg and pegasus will convert it for you and varnish will cache it.  it's worth trying on photographic-style images just to see if we can save some significant bandwidth.

File: pegasus/sites.v3/hourofcode.com/layouts/wide.haml [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
 
 .container
 
   - if header['nav']
     .row
-      .col-sm-3{style: "margin-top:50px; background-color: rgba(0,0,0,0.02); font-size: 16px; padding-left: 0px; padding-right: 0px;"}
-        = view(header['nav'])
       .col-sm-9
         = body
+      .col-sm-3{style: "margin-top:50px;"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/layouts/wide.haml]: Could we also scoop up this in-line css into wide-layout.css?

File: pegasus/sites.v3/hourofcode.com/public/css/wide-layout.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
+.display-link {
+	border-radius: 5px;
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/css/wide-layout.css]: This file is using tab indents instead of 2 space indent.

Comment (caleybrock) [inline pegasus/sites.v3/hourofcode.com/public/css/wide-layout.css]: Ah sorry, I'll fix all of these. I have to figure out why sublime keeps doing this.

File: pegasus/sites.v3/hourofcode.com/views/header.haml [hunk 4]
```diff
@@ -14,7 +14,8 @@
       #language-dropdown.hidden-xs{style: "visibility: hidden;"}
         = view :language_dropdown
       #right.hidden-xs
-        %a.headerlink{href:resolve_url('/resources'), style: "margin-left:0px"}= hoc_s(:header_menu_resources)
+        %a.headerlink{href:resolve_url('/how-to'), style: "margin-left:0px"} How-To
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/header.haml]: These 2 should be localized in hoc.com/i18n/en.yml

File: pegasus/sites.v3/hourofcode.com/views/how_to_nav.md [hunk 5]
```diff
@@ -1,13 +1,17 @@
-- <a href="<%= resolve_url('/resources') %>"><%= hoc_s(:nav_back_to_resources)%></a>
-- <a href="<%= resolve_url('/resources/how-to') %>"><%= hoc_s(:how_to_nav_educators)%></a>
-- <a href="<%= resolve_url('/resources/how-to-public-officials') %>"><%= hoc_s(:how_to_nav_public_officials)%></a>
-- <a href="<%= resolve_url('/resources/how-to-districts') %>"><%= hoc_s(:how_to_nav_districts)%></a>
-- <a href="<%= resolve_url('/resources/how-to-events') %>"><%= hoc_s(:how_to_nav_assemblies)%></a>
-- <a href="<%= resolve_url('/resources/how-to-organizations') %>"><%= hoc_s(:how_to_nav_organizations)%></a>
+<div class="resources-navigation">
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/how_to_nav.md]: I think we should make these navs haml files. It's looking pretty messy with all the tags. Plus anything in /views is really saying it should be haml.

File: pegasus/sites.v3/hourofcode.com/views/promote_nav.md [hunk 6]
```diff
@@ -1,13 +1,17 @@
...
-- <a href="<%= resolve_url('/resources/promote#videos') %>"><%= hoc_s(:resources_videos)%></a>
-- <a href="<%= resolve_url('/resources/promote#posters') %>"><%= hoc_s(:resources_posters)%></a>
-- <a href="<%= resolve_url('/resources/promote#banners') %>"><%= hoc_s(:resources_banners)%></a>
-- <a href="<%= resolve_url('/resources/promote#stickers') %>"><%= hoc_s(:resources_stickers)%></a>
-- <a href="<%= resolve_url('/resources/promote#sample-emails') %>"><%= hoc_s(:resources_emails)%></a>
-- <a href="<%= resolve_url('/resources/stats') %>"><%= hoc_s(:resources_stats)%></a>
-- <a href="<%= resolve_url('/resources/press-kit') %>"><%= hoc_s(:resources_press_kit)%></a>
+<div class="resources-navigation">
+	<div class="nav-header"><%= hoc_s(:resources_promote)%></div>
+	<div class="nav-item"><a class="nav-link" href="<%= resolve_url('/promote#onepager') %>"><%= hoc_s(:resources_one_pager)%></a></div>
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/promote_nav.md]: And as a haml file it'd probably be really clean to have a list of all the pages and just substitute they key word like "brochure" in the url and the string.

File: pegasus/sites.v3/hourofcode.com/views/resources_nav.haml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+      %div.link-text.display-purple= hoc_s(:resources_nav_promote)
+  %a{href: resolve_url('https://code.org/learn')}
+    %div.display-link
+      %img.display-img{src: "/images/try_tutorials.png"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/resources_nav.haml]: I haven't done a blanket change for all of hoc.com, but if you touch files could you check that all links, files, and images are localized?

    src: localized_image("/images/try_tutorials.png"}

**Comment by caleybrock** [comment]
Made fixes for all comments. Now fixing mobile ordering.

**Comment by caleybrock** [comment]
For now, for responsive, all the navigation goes before the content (so there is just lots of scrolling). I'll do a follow up PR with the solution that we talked about, but it's more complicated and I don't want to block. 

@tanyaparker @breville can either of you do a final review?

**Comment by breville** [comment]
lgtm, though from looking through the source, not looking at final pages in browser.

interestingly, 27 commits seems like a lot.. you don't have to commit every single change separately, but we can chat more about this philosophy tomorrow :)

**Pull request merged**